report_qweb_signer/models/report.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingenieria S.L. - Antonio Espinosa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import base64
+from contextlib import closing
+import os
+import subprocess
+import tempfile
+import time
+
+from openerp import SUPERUSER_ID, models, api, _
+from openerp.exceptions import Warning, AccessError
+from openerp.tools.safe_eval import safe_eval as eval
```

Comment:
please don't rename safe_eval: it makes looking for unsafe 'raw' eval calls harder than needed when reviewing the code.